src/Uno.UI/Controls/CommandBar/AppBarButtonRenderer.Android.cs

```diff
@@ -152,14 +152,13 @@ protected override void Render()
 			native.SetVisible(element.Visibility == Visibility.Visible);
 
 			// Foreground
-			var foreground = (element.Icon?.Foreground) as SolidColorBrush;
-
-			var foregroundOpacity = foreground?.Opacity ?? 0;
+			var foregroundOpacity = 0d;
 			if (native.Icon != null)
 			{
-				if (foreground != null)
+				if (element.TryGetIconColor(out var iconColor))
 				{
-					DrawableCompat.SetTint(native.Icon, (Android.Graphics.Color)foreground.Color);
+					foregroundOpacity = iconColor.A / 255d;
+					DrawableCompat.SetTint(native.Icon, (Android.Graphics.Color)iconColor);
```

Comment:
We should be able to UI tests those changes, right? Runtime should be ok now as well.